src/couch_replicator_httpc.erl

```diff
@@ -162,7 +167,11 @@ process_stream_response(ReqId, Worker, HttpDb, Params, Callback) ->
     receive
     {ibrowse_async_headers, ReqId, Code, Headers} ->
         case list_to_integer(Code) of
+        429 ->
+            Timeout = couch_replicator_ratelimiter:limit(),
```

Comment:
I suggest `couch_replicator_rate_limiter` is a better module name, and the flip between using the word 'limit' and the result being a Timeout is weird. `couch_replicator_rate_limit:calculate_timeout()`?